src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -29,7 +29,7 @@
 namespace System.Management.Automation
 {
     /// <summary>
-    /// 
+    ///
```

Comment:
Please avoid formatting changes in a PR with meaningful code changes - it makes reviewing the code differences harder.